### PR TITLE
Rework getting started section

### PR DESCRIPTION
**Pull request by philippmwirth**
# Rework getting started section

Adds instructions to
- Install the PIP package
- Run the docker in "worker mode"
- Minimal restructuring

PDF: [setup.pdf](https://github.com/lightly-ai/lightly/files/8830678/setup.pdf)

### CODE REVIEW

**Review by guarin — APPROVED** [review]
Awesome!

File: docs/source/docker/getting_started/setup.rst [hunk 1]
```diff
@@ -101,12 +115,33 @@ Create a new docker tag using the following command:
 .. note:: If you don't want to tag the image name you can replace lightly/worker:latest
           by eu.gcr.io/boris-250909/lightly/worker:latest for all commands in this documentation.
 
+
+
+Update Lightly Docker
+^^^^^^^^^^^^^^^^^^^^^
```
Comment (guarin) [inline docs/source/docker/getting_started/setup.rst:121]: Inconsistent `Lightly Docker` with line 158 where `Lightly Worker` is used.

**Review by guarin — CHANGES_REQUESTED** [review]
There might be some more worker <-> docker inconsistencies. Maybe quickly verify by searching for `docker`.

File: docs/source/docker/getting_started/setup.rst [hunk 2]
```diff
@@ -119,20 +154,45 @@ You should see an output similar to this one:
...
+.. code-block:: python
+
+    # execute the following code once to get a worker_id
+    from lightly.api import ApiWorkflowClient
+
+    client = ApiWorkflowClient(token='MY_AWESOME_TOKEN') # replace this by your token
+    worker_id = client.register_compute_worker()
+    print(worker_id)
+
+Store the `worker_id` in a secure location and then start the docker in worker mode with
```
Comment (IgorSusmelj) [inline docs/source/docker/getting_started/setup.rst:178]: Maybe add a link to the web app to see the list of registered workers? (https://app.lightly.ai/compute/workers)

**Review by IgorSusmelj — APPROVED** [review]
One small suggestion. Otherwise, fine for me :)

**Review by guarin — COMMENTED** [review]
The docker/worker naming still seems a bit inconsistent. For example the `Analytics` and the `Licensing` sections still use `docker`. 

I would suggest the following:
* Use worker by default
* Only use docker when it is important that it is a docker image (for example in the docker pull/tag commands)
* Mention somewhere that the worker is simply a docker container.

**Review by guarin — APPROVED** [review]

**Pull request merged**